src/main/java/org/terasology/minerals/generation/OreRasterizer.java

```diff
@@ -69,51 +70,52 @@ public void generateChunk(CoreChunk chunk, Region chunkRegion) {
     }
     
     protected void generateChunk(CoreChunk chunk, Region chunkRegion, Block ore, OreFacet oreFacet) {
-        for(Vector3i position : chunkRegion.getRegion()) {
+        for(Vector3ic pos : chunkRegion.getRegion()) {
+            Vector3i position = new Vector3i(pos);
```

Comment:
not sure whether we need this, could also use temp vector for calcRelativeBlockPos